quetz/tasks/workers.py

```diff
@@ -140,3 +139,65 @@ async def wait(self):
         loop = asyncio.get_event_loop()
         if self.future:
             return await loop.run_in_executor(None, self.future.result)
+
+
+class RQManager(AbstractWorker):
+    def __init__(self, host, port, db, api_key: str, browser_session: dict):
+        self.host = host
+        self.port = port
+        self.db = db
+        self.api_key = api_key
+        self.browser_session = browser_session
+        self.conn = redis.StrictRedis(host=self.host, port=self.port, db=self.db)
+        self.queue = Queue(connection=self.conn)
+
+    @staticmethod
+    def wrapper(self, func, api_key, browser_session, **kwargs):
```

Comment:
static methods don't take the self, argument so it should rather be:

```suggestion
    def wrapper(func, api_key, browser_session, **kwargs):
```

Reply:
Yes, but even removing this doesn't work. See the above comment.